bears/vcs/git/GitCommitBear.py

```diff
@@ -190,11 +192,16 @@ def check_body(self, body,
             return
 
         if body[0] != "":
-            yield Result(self, "No newline between shortlog and body at HEAD.")
+            yield Result(self, "No newline found between shortlog and body at"
+                               "HEAD commit. Please add one.")
             return
 
         ignore_regexes = [re.compile(regex) for regex in ignore_length_regex]
         if any((len(line) > body_line_length and
                 not any(regex.search(line) for regex in ignore_regexes))
                for line in body[1:]):
-            yield Result(self, "Body of HEAD commit contains too long lines.")
+            yield Result(self, "Body of HEAD commit contains too long lines. "
+                               "Commit body lines should not be over {} "
+                               "characters, as Github wraps them up and those "
+                               "characters would be lost.".format(
```

Comment:
You can adjust the length, reasoning this using Github's standards isn't very objective^^